src/main/java/net/gudenau/minecraft/moretags/mixins/block/farmland/CropBlockMixin.java

```diff
@@ -46,10 +46,10 @@ private static boolean isBlockFarmland(BlockState blockState, Block block) {
     
     @Inject(
         method = "canPlantOnTop",
-        at = @At("HEAD"),
+        at = @At("RETURN"),
         cancellable = true
     )
     private void canPlantOnTop(BlockState floor, BlockView world, BlockPos pos, CallbackInfoReturnable<Boolean> cir) {
-        cir.setReturnValue(floor.isIn(MoreBlockTags.FARMLAND));
+        cir.setReturnValue(cir.getReturnValue() || floor.isIn(MoreBlockTags.FARMLAND));
```

Comment:
```suggestion
if (!cir.getReturnValue() && floor.isIn(MoreBlockTags.FARMLAND)) {
        cir.setReturnValue(true);
}
```

Reply:
Is this to let other mods deny certain blocks?

Reply:
That would prevent you from removing farmland blocks.